packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -1546,3 +1534,13 @@ ${_argParser.usage}''';
     }
   }
 }
+
+extension on dart_ast.NamedType {
```

Comment:
is this temporary or permanent?

if it's temporary, please add a TODO explaining the timeline for when it will be removed

if it's permanent, please use a function instead of an extension. The use of Dart extension features violates our style guide due to its harm on readability (someone reading the code will look for "qualifiedName" on the `dart_ast.NamedType` API and not find it; there's nothing to point them to this part of the code).